tests/keras/test_sequential_model.py

```diff
@@ -8,9 +8,21 @@
 from keras import backend as K
 import keras
 from keras.models import Sequential
-from keras.layers import Dense, Activation
+from keras.layers import Dense
+from keras.layers import Activation
+from keras.layers import BatchNormalization
+from keras.layers import Conv2D
+from keras.layers import Conv3D
+from keras.layers import Dropout
+from keras.layers import ReLU
+from keras.layers import UpSampling3D
+from keras.layers import Lambda
+from keras.layers import SimpleRNN
+from keras.layers import Softmax
```

Comment:
Rather than lots of such imports, prefer importing `layers` then using e.g. `layers.Dense`.